app/uk/gov/hmrc/agentservicesaccount/views/pages/agent_services_account.scala.html

```diff
@@ -90,7 +90,7 @@ <h3 class="heading-small card-heading">
 
         <div class="column-one-half">
 
-            @if(configuration.getBoolean("features.showAgentInvitationsLink").getOrElse(false) || configuration.getBoolean("features.showAgentMappingLink").getOrElse(false)){
+            @if(configuration.getBoolean("features.showAgentInvitationsLink").getOrElse(false) || configuration.getBoolean("features.showAgentMappingLink").getOrElse(false) || configuration.getBoolean("features.showAgentLedDeAuthLink").getOrElse(false)){
```

Comment:
Just an aside - shouldn't `features.showAgentInvitationsTrackLink` also be in this if condition?

Reply:
not sure why its not there , but i can add it

Reply:
just added !